application/src/test/java/org/opentripplanner/updater/trip/siri/ModifiedTripBuilderTest.java

```diff
@@ -424,132 +434,6 @@ void testUpdateUpdatedStop() {
     assertEquals(RealTimeState.MODIFIED, updatedTimes.getRealTimeState());
   }
 
-  @Test
-  void testUpdateCascading() {
```

Comment:
These tests just test something that we have now removed, right?

Reply:
Yes, `testUpdateCascading` assumes that we accept a message with fewer calls than in the scheduled trip. We now refuse these messages except for full cancellation, and full cancellation is tested elsewhere.
`testCreateStopPatternNoCalls` and `testCreateStopPatternSingleCall` call `ModifiedTripBuilder.createStopPattern` with preconditions that are now impossible.